Source/SwiftLintCore/Models/Issue.swift

```diff
@@ -20,6 +20,22 @@ public enum Issue: LocalizedError, Equatable {
     /// Used rule IDs are invalid.
     case invalidRuleIDs(Set<String>)
 
+    /// Found a rule configuration for a rule that is not present in `only_rules`.
+    case configurationForRuleNotPresentInOnlyRules(ruleID: String)
```

Comment:
Any reason this uses a different naming style?
```suggestion
    case ruleNotPresentInOnlyRules(ruleID: String)
```